ambari-server/src/main/java/org/apache/ambari/server/state/Cluster.java

```diff
@@ -95,7 +95,9 @@ Service addDependencyToService(String  serviceGroupName, String serviceName,
    * @return
    * @throws AmbariException
    */
-  ServiceGroup addServiceGroup(String serviceGroupName) throws AmbariException;
+  ServiceGroup addServiceGroup(String serviceGroupName, String stackId) throws AmbariException;
```

Comment:
Rename "String stackId" to "String version" in the interface.

Reply:
The change is already there.

Reply:
ok, changed it.